### ISSUE TITLE
FTP upload - does not create folders on v1.0.0-alpha1

### ISSUE THREAD

**Issue by ay-git** [Altinity/clickhouse-backup#164]
Before uploading files to FTP need create a folder structure.
clickhouse-backup try immediately upload the file and gets an error:
553 Could not create file.

Debug log:
```
clickhouse-backup upload ch_backup_20210312_ok2
220 (vsFTPd 3.0.3)
FEAT
211-Features:
 EPRT
 EPSV
 MDTM
 PASV
 REST STREAM
 SIZE
 TVFS
211 End
USER stat3
331 Please specify the password.
PASS gdfj549b_mHRH
230 Login successful.
TYPE I
200 Switching to Binary mode.
PASV
227 Entering Passive Mode (172,104,252,28,223,107).
LIST ch_backup_20210312_ok2/shadow/stat_new/stat_events
150 Here comes the directory listing.
226 Transfer done (but failed to open directory).
 0 / 23107840518 [---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------]   0.00%PASV
227 Entering Passive Mode (172,104,252,28,59,230).
STOR ch_backup_20210312_ok2/shadow/stat_new/stat_events/data2_1.tar
553 Could not create file.
 3.06 MiB / 21.52 GiB [>------------------------------------------------------------------------------------------------------------------------------------------------------------------------]   0.01% 0s
2021/03/13 18:17:40 error can't upload: 553 Could not create file.
```

**Comment by Camamber** [comment]
While try ftp upload it says

```
2021/03/23 12:50:30 error can't upload: 550 /clickhouse/clickhouse_move_2021-03-22T12-49-20/shadow/default/table2: No such file or directory
``` 

But if i create this folders manual its working, but only for created directory

**Comment by AlexAkulov** [comment]
Fixed in [v1.0.0-beta1](https://github.com/AlexAkulov/clickhouse-backup/releases/tag/v1.0.0-beta1)
